tests/bluetooth/tester/src/btp_bap_stream.h

```diff
@@ -0,0 +1,216 @@
+/* btp_bap_stream.h - Bluetooth BAP Tester */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef TESTER_BTP_BAP_STREAM_H
+#define TESTER_BTP_BAP_STREAM_H
+
+#include <zephyr/bluetooth/audio/cap.h>
+
+#define SERVER_SNK_COUNT CONFIG_BT_ASCS_ASE_SNK_COUNT
+#define SERVER_SRC_COUNT CONFIG_BT_ASCS_ASE_SRC_COUNT
+#define CLIENT_SNK_COUNT CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT
+#define CLIENT_SRC_COUNT CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT
+#define MAX_SNK_STREAMS_COUNT MAX(CLIENT_SNK_COUNT, SERVER_SNK_COUNT)
+#define MAX_SRC_STREAMS_COUNT MAX(CLIENT_SRC_COUNT, SERVER_SRC_COUNT)
```

Comment:
This looks a bit weird. 

Why are we mixing client and server configs here?

Wouldn't the maximum streams supported also be the minimum of those 2? 

e.g. if `CLIENT_SNK_COUNT=2` and `SERVER_SNK_COUNT=4`, can't we only support 2 sink streams?

Reply:
> Why are we mixing client and server configs here?
Because of the `struct btp_bap_unicast_connection` that is used for both client and server. I might split this too in the future patches.